support-config/src/main/scala/com/gu/support/config/StripeConfig.scala

```diff
@@ -1,24 +1,47 @@
 package com.gu.support.config
 
-import com.gu.i18n.Currency
+import com.gu.i18n.{Country, Currency}
 import com.gu.i18n.Currency.AUD
 import com.gu.monitoring.SafeLogger
 import com.typesafe.config.Config
 
 
-case class StripeConfig(defaultAccount: StripeAccountConfig, australiaAccount: StripeAccountConfig, version: Option[String] = None)
+case class StripeConfig(defaultAccount: StripeAccountConfig,
+                        australiaAccount: StripeAccountConfig,
+                        unitedStatesAccount: StripeAccountConfig,
+                        version: Option[String])
```

Comment:
I removed the Option[String] = None here as I tripped up on it and accidentally deleted some code that passed a value into it.

Reply:
Doesn't `= None` just provide an initial value in the case that a value isn't passed in?

Reply:
Is it that it's easier to read without it?

Reply:
Yup, but it also means you can make the mistake I did before this commit: https://github.com/guardian/support-frontend/pull/2040/commits/ef28ff44d181fc1a63dd90e9039d6549e6ba4de2

Reply:
Ah so it's because `None` isn't actually something we ever want there, so it's a bad default parameter

Reply:
Not quite, let's chat f2f tomorrow.